cli/internal/terraform/terraform/iam/aws/main.tf

```diff
@@ -232,11 +226,13 @@ resource "aws_iam_role_policy_attachment" "attach_bootstrapper_policy_control_pl
   policy_arn = aws_iam_policy.constellation_bootstrapper_policy.arn
 }
 
+// TODO(msanft): incorporate this into the custom worker node policy
 resource "aws_iam_role_policy_attachment" "csi_driver_policy_worker" {
   role       = aws_iam_role.worker_node_role.name
   policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy"
 }
 
+// TODO(msanft): incorporate this into the custom worker node policy
```

Comment:
```suggestion
// TODO(msanft): incorporate this into the custom control-plane node policy
```

Reply:
By "incorporate into custom policy", do you mean to extract the permissions from the managed AWS-CSI role and add the permissions to our role?

Reply:
Exactly. I think we should add this to the minimal permission set we define for all CSPs

Reply:
The one that we document, that's used in the CI, etc.

Reply:
Do you think it is something worth doing because the managed role provides excess permissions, or is it just to combine permissions into one?
If its the later I would argue against it and keep the current way of attaching the role, as this guarantees the role to always have the correct permissions for CSI